src/components/menu.tsx

```diff
@@ -218,7 +218,12 @@ export const Menu = ({ onHide, visible }: Props) => {
               </MenuItem>
             ) : (
               <>
-                <MenuItem icon={login} href="/login">
+                <MenuItem
+                  icon={login}
+                  href={`/login?from=${encodeURIComponent(
+                    window.location.href,
```

Comment:
```suggestion
                    location.pathname,
```

It should still work, but with this change the query parameter will look a little bit nicer without the domain